### DISCUSSION TITLE
after closing NON-active tab, do ...

### DISCUSSION THREAD

**Discussion by Elementari** [mbnuqw/sidebery#912] [Ideas]
v4

Hello, in v4, in settings, there is a settings where the user can choose that after closing a currently ACTIVE tab, a certain action will be performed, such as in particular: show previously active tab, or "do nothing".
However it seems to me that when I close ANY tab (not just the currently active one), this particular action will be performed.
What I would want is that when closing the currently ACTIVE tab, the previously active tab should be shown, but when closing a NON active tab, NOTHING should be done.
the current situation is particularly problematic with hundreds of tabs, where closing a tab that is not active will subsequently cause the list of tabs to jump to some random place where some other tab that happens to have been previously opened is located even though typically the user does NOT intend to go there, but instead was in the process of closing a bunch of tabs.
whereas when the user is actively surfing, closing the currently active tab will typically be a situation where he will want to procede with the previously active tab.

**Comment by mbnuqw** [comment]
Sounds like a bug (that probably fixed in v5 beta). You can check [beta version](https://github.com/mbnuqw/sidebery/releases/tag/v5.0.0b29) in a [new Firefox profile](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles) to check if this still happening and open an issue if it does.
